utils/cargo-tfhe-lints-inner/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "cargo-tfhe-lints-inner"
+version = "0.1.0"
+edition = "2021"
+
+# A noop dependency that changes in the Rust repository, it's a bit of a hack.
+# See the `src/tools/rustc-workspace-hack/README.md` file in `rust-lang/rust`
```

Comment:
could not find that readme do you know if the rust project still documents this ?

Reply:
maybe this is not needed anymore, I will check that !

Reply:
this is not needed since https://github.com/rust-lang/rust/pull/109133 has been merged